### PR TITLE
133937155 remove harrow community network from seo title tag

### PR DESCRIPTION
**Pull request by kpediad**
https://www.pivotaltracker.com/story/show/133937155

### CODE REVIEW

**Comment by tansaku** [comment]
great work here @kpediad !

File: features/titletagchange_133937155.feature [hunk 1]
```diff
@@ -0,0 +1,27 @@
+Feature: I want to shorten the static part of the title to contain only "Harrow volunteering" instead of "Harrow Community Network | Harrow volunteering"
```
Comment (tansaku) [inline features/titletagchange_133937155.feature]: we wouldn't usually have the story id in the filename here - also I'm not sure we need a separate feature file here - I think the functional change is actually covered by `features/admin/seo.feature` so we could perhaps remove this one ...

**Comment by kpediad** [comment]
@tansaku can you please provide a bit of guidance here? I did remove the feature file and two new step definitions that I had added to be used from the feature file that was removed. rspec and cucumber tests gave me all greens on my development environment before I pushed the changes to github. Travis gave a failure in a test that I have not really touched upon. Any idea what is happening?

**Comment by tansaku** [comment]
@kpediad excellent - and now the PR becomes an easy merge, easy to review - wonderful; thakyou so much!

Only one more pieces of feedback - if we had [Finishes #133937155] in the commit message as specified in CONTRIBUTING.md https://github.com/AgileVentures/LocalSupport/blob/develop/CONTRIBUTING.md#git-comments the PT story would automatically close once I merge.  I can easily do it manually this time - just something to add on your next PR :-)

**Pull request merged**